astropy/modeling/functional_models.py

```diff
@@ -1754,14 +1758,25 @@ def __init__(
         amplitude_L=amplitude_L.default,
         fwhm_L=fwhm_L.default,
         fwhm_G=fwhm_G.default,
-        method="humlicek2",
+        method=None,
         **kwargs,
     ):
+        if method is None:
+            if HAS_SCIPY:
+                method = "wofz"
+            else:
+                method = "humlicek2"
+
         if str(method).lower() in ("wofz", "scipy"):
             from scipy.special import wofz
 
             self._faddeeva = wofz
         elif str(method).lower() == "humlicek2":
+            warnings.warn(
+                f"{method} has been depricated in favor of the `wofz` method which requires `scipy`",
```

Comment:
```suggestion
                f"{method} has been deprecated since Astropy 5.3 and will be removed in a future version. "
                "It is recommended to always use the `wofz` method which requires `scipy`.",
```
Should put the actual versions here once the roadmap for this is settled.